content/en/continuous_testing/troubleshooting.md

```diff
@@ -26,11 +28,18 @@ Batch timeout
 Execution rule
 : An [execution rule][4] defines the impact of a test failure on a CI pipeline from most to least impactful: `skipped`, `non_blocking`, and `blocking`. These options are weighted and defaults to the most impactful. If your test is configured as `skipped` in the UI and `blocking` in the configuration file, it is skipped during the test run. </br><br> You can set the execution rule in your tests' properties, global configuration file, or an individual test's override file. 
 
+## CI Results Explorer
+
+### CI metadata does not appear
+
+Check whether you are using API endpoints to trigger your CI/CD test runs. In order to populate the CI Results Explorer with CI metadata, you must use the [NPM package][5].
```

Comment:
```suggestion
Check whether you are using API endpoints to trigger your CI/CD test runs. In order to populate the CI Results Explorer with CI metadata, you must use one of our integrations under [CI/CD integrations][3].
```

Thinking we should point to the GitHub Action, Circle CI orb, Azure DevOps Extension and Jenkins integration, before deferring them to the NPM package.